pkg/cmd/cli/cmd/newapp.go

```diff
@@ -166,6 +166,8 @@ func RunNewApplication(fullName string, f *clientcmd.Factory, out io.Writer, c *
 			}
 		}
 	}
+	fmt.Fprintln(c.Out(), "Run 'oc status' to view your app.")
```

Comment:
Isn't it better to use the `fullName` var instead of explicitly specifying `oc`?